src/Parser/MistweaverMonk/CombatLogParser.js

```diff
@@ -661,6 +661,89 @@ class CombatLogParser extends MainCombatLogParser {
           <LowHealthHealingTab parser={this} />
         ),
       },
+      {
+        title: 'Player Log Data',
+        url: 'player-log-data',
+        render: () => (
+          <div>
+            <div className="panel-heading">
+              <h2>Player Log Data</h2>
+            </div>
+            <div style={{ padding: '15px 22px 15px 15px' }}>
+              <div style={{ padding: '0px 22px 15px 0px' }}>Please use the below table to populate the Player Log section of the Mistweaver Spreadsheet by Garg.  <a href="http://www.peakofserenity.com/mistweaver/spreadsheet/" target="_blank" rel="noopener noreferrer">Link to the sheet</a><br /></div>
+              <div>
+              <table key='playlogdata-heading' style={{borderBottom: '1px solid #dddddd', borderTop: '1px solid #dddddd', align: 'left', padding: '8px', float: 'left', margin: '2px'}}>
+                <tr><td>Fight Length (Minutes)</td></tr>
+                <tr><td>Fight Length (Seconds)</td></tr>
+                <tr style={{borderTop: '.5px solid #dddddd', borderBottom: '.5px solid #dddddd'}}><td>Mana Remaining</td></tr>
```

Comment:
When you have this in a separate file you can clean up the repeated CSS objects by defining an object at the top of the file, like so:

```js
const styles = {
  someClassName: { borderTop: '.5px solid #dddddd', borderBottom: '.5px solid #dddddd' },
};
```

Alternatively you could make a new css file, import this at the top and use CSS classes like you'd do in regular HTML.